### PR TITLE
feat: simplified grid

### PR DESCRIPTION
**Pull request by shashidwivedi**
Simplified Grid

- grid-2: two items in a grid
- grid-3: three items

PS: Do not review foobar-ui.css and foobar-ui.min.css as they are generated files

image

### CODE REVIEW

File: styles/foobar-ui.css [hunk 1]
```diff
@@ -446,6 +449,14 @@ template {
...
 }
 
+.hide {
+  display: none;
+}
+
+.invisible {
+  visibility: hidden;
+}
+
```
Comment (MaruthiPothuganti) [inline styles/foobar-ui.css:459]: These styles are already present in utility CSS. Instead of repeating, you can reuse them.

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Everything else looks good.

**Review by shivangisharma01 — APPROVED** [review]
You can apply the suggestion given earlier by the review, other than that  Looks good to merge.

File: styles/foobar-ui.css (hunk 1 above)
Comment (shashidwivedi) [reply styles/foobar-ui.css:459]: Thanks Maruthi, this is actually a generated CSS file, to be used as a single file when someone will use the component library outside.

**Pull request merged**